boards/arm/adafruit_kb2040/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2021 Pete Johanson
```

Comment:
Not sure if it's important or not, but 2022 (fix everywhere)

Reply:
Fixed.